nix/tests/cabal-fmt.nix

```diff
@@ -0,0 +1,30 @@
+{ runCommand, fixCabalFmt, src, lib, diffutils, glibcLocales }:
+let
+  src' = lib.sourceFilesBySuffix src [ ".cabal" ];
```

Comment:
This is why you need CI and to test things yourself and not trust me :D It's `sourceFilesBySuffices`